src/components/Button/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import { OverridableComponent } from "@material-ui/core/OverridableComponent";
+import { Button as MacawButton, ButtonTypeMap } from "@saleor/macaw-ui";
+import { isExternalURL } from "@saleor/utils/urls";
+import React from "react";
+import { Link } from "react-router-dom";
+
+const _Button: React.FC<any> = React.forwardRef(({ href, ...props }) => {
```

Comment:
Changed the name to just `Button`. If I used named export, I would still need these shenanigans because of type assertion.

We're not using named exports in dashboard (I know, I know) but since that's the existing convention I think it's better to stick to what's already there rather than to have different components using named exports and the other ones using default export.

Reply:
done